test/puppetlabs/ssl_utils/core_test.clj

```diff
@@ -500,6 +502,59 @@
             (is (revoked? updated-crl cert))
             (is (revoked? updated-crl cert2))))))))
 
+(deftest validate-cert-chain-and-crls-test
+ (testing "validate cert chain against CRLs"
```

Comment:
I realize I didn't add test cases around invalid certs (just invalid CRLs). While that was originally motivated by my needs for this ticket (since the incoming CRLs are the only piece we won't have control over), I'm thinking it might be okay, since we'd kinda just be testing the Java libs we're using? Thoughts?